CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Misc:
 
 Breaking changes:
 - [#1662](https://github.com/rails-api/active_model_serializers/pull/1662) Drop support for Rails 4.0 and Ruby 2.0.0. (@remear)
+- [#1720](https://github.com/rails-api/active_model_serializers/pull/1720) Block relationships must explicitly use `load_data` instead of the return value to customize relationship contents.
```

Comment:
Unfortunately `load_data` is required when you opt-in to this feature (see https://github.com/rails-api/active_model_serializers/pull/1720#issuecomment-217910604 for explanation).  I could make it so when you opt-out it's still the old API, but this seems confusing to me.

My hunch is you'll probably want this API (running the relationship block but lazy-loading the data) for other features as well, though I can't think of any top of my head.